### DISCUSSION TITLE
記事ページの投稿日と編集履歴ページの投稿日が一致していない記事がある

### DISCUSSION THREAD

**Discussion by munierujp** [increments/qiita-discussions#328] [要望・アイデア・不具合]
https://qiita.com/suin/items/81d25593046556f7f4ef

たとえば上記の記事について、記事ページの投稿日は「2012年07月31日」となっていますが、 編集履歴ページでは「2012年12月26日」となっています。

https://qiita.com/suin/items/81d25593046556f7f4ef/revisions/0

これはなぜでしょうか？
また、どちらを信用すればよいでしょうか？

**Comment by wataru86** [comment]
@munierujp 
混乱させてしまい申し訳ありません。
記事ページに表示されている「2012年07月31日」が正しいです。
編集履歴機能の追加が「2012年12月26日」で、それよりも前に投稿された記事の編集履歴ページでは投稿が「2012年12月26日」となっています 🙇

**Reply by munierujp** [reply]
なるほど、そういうことだったのですね。
ありがとうございます。
